tests/telemetry/telemetry_test.go

```diff
@@ -0,0 +1,174 @@
+package tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	torpedovolume "github.com/portworx/torpedo/drivers/volume"
+	. "github.com/portworx/torpedo/tests"
+)
+
+func TestTelemetryBasic(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_telemetry.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Telemetry", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+// This test performs basic test of starting an application and destroying it (along with storage)
+var _ = Describe("{DiagsBasic}", func() {
+	var contexts []*scheduler.Context
+	It("has to setup, validate, try to get diags on nodes and teardown apps", func() {
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("setupteardown-diags-%d", i))...)
```

Comment:
Change `setupteardown-diags` to `diagsbasic`. This will make it easier to identify the k8s namespace name with the focus tests that we select in Jenkins (The focus test will be DiagsBasic)